tests/sap/aibus/dar/client/test_dar_session.py

```diff
@@ -251,3 +253,88 @@ def test_plain_http_is_not_allowed(self):
             "URL must use https scheme. Unencrypted connections" " are not supported."
         )
         assert str(context.value) == expected_message
+
+
+@pytest.fixture
+def dar_url():
+    return "https://localhost/"
+
+
+@pytest.fixture
+def credentials_source():
+    return StaticCredentialsSource("12345")
+
+
+@pytest.fixture
+def dar_session_500_error(dar_url, credentials_source):
```

Comment:
Not sure if you are familiar with this syntax: pytest will automatically provide the output of the fixtures in line 258 and 263.

Reply:
Thanks for the info